Packs/ServiceNow/Integrations/ServiceNowv2/README.md

```diff
@@ -94,6 +94,9 @@ If MFA is enabled for your user, follow the next steps:
 | update_timestamp_field | Timestamp field to query for updates as part of the mirroring flow. | False |
 | mirror_limit | The maximum number of incidents to mirror incoming each time | False |
 | close_incident | Close XSOAR Incident. When selected, closing the ServiceNow ticket is mirrored in Cortex XSOAR. | False |
+| close_ticket_multiple_options | Define the closing method for XSOAR Incident. When selected, closing / resolving the ServiceNow ticket is mirrored in Cortex XSOAR. /
```

Comment:
can we make the decription here the same as in the yml?

Reply:
Bar asked me to put more detailed information about pre-configured integrations somewhere.
Is there anywhere else you suggest putting this info?

Reply:
I think releasenotes would probably be a good place then